lib/adminRoutes.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+export const adminRoutes: string[] = [
```

Comment:
Why can't this be a wildcard? `admin/*`? Then, we don't need to track every page.

Reply:
✅ done